packages/frontend/src/providers/AppProvider.tsx

```diff
@@ -182,13 +183,26 @@ export const AppProvider: FC = ({ children }) => {
         },
         [showToastSuccess],
     );
+
+    const showToastInfo = useCallback<AppContext['showToastInfo']>(
+        (props) => {
+            showToastSuccess({
+                intent: Intent.NONE,
+                icon: 'refresh',
```

Comment:
Should we have a generic icon for the info toasts ? and we can set the refresh icon in the RefreshServer component.